testing-utils/src/lib/setup/console-log.mock.ts

```diff
@@ -0,0 +1,11 @@
+import { SpyInstance, afterEach, beforeEach, vi } from 'vitest';
+
+let consoleLogSpy: SpyInstance;
+
+beforeEach(() => {
+  consoleLogSpy = vi.spyOn(console, 'log');
+});
+
+afterEach(() => {
+  consoleLogSpy.mockRestore();
+});
```

Comment:
This should be changed to spy on `console.info`, `console.warn` and `console.error`, but not `console.log`. Otherwise debugging tests will be very difficult, because nothing will be printed.

But we 1st need to replace all our intentional `console.log`s to `console.info` (and turn on the `no-console` ESLint rule). I would suggest this be prioritized very soon. Perhaps even before this is merged?

Reply:
This will be tackled in issue #277 and it affects all packages so I wouldn't do it in this PR which only targets the CLI package. But I can prioritise that issue next 👍